markets/perps-market/contracts/storage/PerpsMarketFactory.sol

```diff
@@ -93,4 +93,24 @@ library PerpsMarketFactory {
     function withdrawMarketUsd(Data storage self, address to, uint256 amount) internal {
         self.synthetix.withdrawMarketUsd(self.perpsMarketId, to, amount);
     }
+
+    function calculateMinimumCredit() internal view returns (uint256 accumulatedMinimumCredit) {
+        SetUtil.UintSet storage activeMarkets = GlobalPerpsMarket.load().activeMarkets;
+        uint256 activeMarketsLength = activeMarkets.length();
+        for (uint i = 1; i <= activeMarketsLength; i++) {
+            uint128 marketId = activeMarkets.valueAt(i).to128();
+
+            accumulatedMinimumCredit += PerpsMarket.requiredCredit(marketId);
+        }
+    }
+
+    function utilizationRate(Data storage self) internal view returns (uint256 utilization) {
+        uint256 withdrawableUsd = self.synthetix.getWithdrawableMarketUsd(self.perpsMarketId);
```

Comment:
I thought this guy was the right method to call to get the delgegatedCollateralValue directly?
    /**
     * @notice Returns the total snxUSD value of the collateral for the specified market.
     * @param marketId The id of the market whose collateral is being queried.
     * @return valueD18 The market's total snxUSD value of collateral, denominated with 18 decimals of precision.
     */
    function getMarketCollateral(uint128 marketId) external view returns (uint256 valueD18);